### PR TITLE
[NG] Go-Live Updates-2

### LINKED ISSUE #1505: [NG] GetNighthawk Project Page
**Current Behavior**
The Projects section does not have GetNighthawk listed yet on layer5-ng website. No page is present to describe the GetNighthawk project on the layer5-ng website.

**Desired Behavior**
Add the GetNighthawk project page as per the following design: https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=2650%3A4242

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

This PR fixes #1505  #1556 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Projects/GNH/index.js [hunk 1]
```diff
@@ -185,6 +186,12 @@ const Projects = () => {
           </Row>
         </div>
       </Container>
+      <div className="callout">
+        <img src={cncf} />
+        <p> Participate in the state of the art. <br/>
+        Join us in the Cloud Native Computing Foundation's Service Mesh Working Group.</p>
+        <Button primary title="Join Us" />
```
Comment (Jashpatel1) [inline src/sections/Projects/GNH/index.js:193]: What url should this button point to?

Comment (leecalcote) [reply src/sections/Projects/GNH/index.js:193]: This one will work well - /community/calendar#meetings

Comment (Jashpatel1) [reply src/sections/Projects/GNH/index.js]: Done

**Comment by Jashpatel1** [comment]
@leecalcote Only the logo is missing.

**Review by leecalcote — APPROVED** [review]
Thank you for knocking out the news item, too, @Jashpatel1 👍

**Pull request merged**